samples/samples-csharp/TriggerBindingSamples/ReservedPrimaryKeyColumnNamesTrigger.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Extensions.Sql.Samples.Common;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Samples.TriggerBindingSamples
+{
+    public static class ReservedPrimaryKeyColumnNamesTrigger
+    {
+        [FunctionName(nameof(ReservedPrimaryKeyColumnNamesTrigger))]
+        public static void Run(
+            [SqlTrigger("[dbo].[ProductsWithReservedPrimaryKeyColumnNames]", ConnectionStringSetting = "SqlConnectionString")]
+            IReadOnlyList<SqlChange<Product>> _)
+        {
+            throw new NotImplementedException();
```

Comment:
For these ones that aren't expected to ever trigger the function let's include a message about this not being valid so it's easy to tell why the test is failing if that happens. 

Something like `throw new NotImplementedException("Test should fail before function code is ran");`

(do this for all similar tests below)